itou/job_applications/notifications.py

```diff
@@ -62,6 +62,26 @@ class JobApplicationNewForEmployerNotification(EmployerNotification, EmailNotifi
     body_template = "apply/email/new_for_employer_body.txt"
 
 
+@notifications_registry.register
+class JobApplicationPostponedForJobSeekerNotification(JobSeekerNotification, EmailNotification):
+    """Notification sent to job seeker when postponed"""
+
+    name = "Mise en attente de candidature"
+    category = NotificationCategory.JOB_APPLICATION
+    subject_template = "apply/email/postpone_for_job_seeker_subject.txt"
+    body_template = "apply/email/postpone_for_job_seeker_body.txt"
+
+
+@notifications_registry.register
+class JobApplicationPostponedForProxyNotification(PrescriberOrEmployerNotification, EmailNotification):
+    """Notification sent to proxy (prescriber or employer/orienter) when postponed"""
+
+    name = "Mise en attente d’une candidature envoyée"
```

Comment:
```suggestion
    name = "Mise en attente d’une candidature"
```
?

Je ne suis pas convaincu de l'ajout de "envoyée", je trouve que ça prête à confution plus qu'autre chose (et dans l'absolu, lorsqu'un candidat soumet elle est aussi "envoyée").
Il n'y a pas de risque de conflit, les deux notifications sont mutuellement exclusives.

Reply:
Le `envoyée` est surtout là pour les employeurs orienteurs qui ont des candidatures envoyées et reçues :grimacing: